### PR TITLE
[Place] landing page and misc fixes

### PR DESCRIPTION
**Pull request by beets**
In addition to the new landing page for place explorer:
* standardize page title across modules
* ranking route and title fix
* search page css fix

image

### CODE REVIEW

File: server/templates/place_landing.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+  <div class="search border mb-4">
+    <div id="location-field">
+      <div id="search-icon"></div>
+      <input id="place-autocomplete"
+        placeholder="Enter a country, state, county or city" type="text" />
+    </div>
+  </div>
+
+  <p>The Place Explorer tool helps you understand the core data
```
Comment (shifucun) [inline server/templates/place_landing.html:48]: is </p> needed here?

File: server/templates/place_landing.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      <a
+        href="{{ ranking_url('UnemploymentRate_Person', 'City', 'country/USA', unit='%') }}">by
+        city</a>,
+      <a
+        href="{{ ranking_url('UnemploymentRate_Person', 'County', 'country/USA', unit='%') }}">by
+        county</a>,
+      <a
+        href="{{ ranking_url('UnemploymentRate_Person', 'State', 'country/USA', unit='%') }}">by
+        state</a>
+    <li><strong>Poverty in the United States</strong>:
```
Comment (shifucun) [inline server/templates/place_landing.html:110]: remove "the" for consistency?

Comment (beets) [reply server/templates/place_landing.html]: done

File: server/templates/place_landing.html (hunk 1 above)
Comment (beets) [reply server/templates/place_landing.html:48]: done, and closed all the other tags as well

**Review by shifucun — DISMISSED** [review]

**Review by shifucun — APPROVED** [review]

**Pull request merged**